app/reducers/devices.js

```diff
@@ -167,6 +168,20 @@ const devices = {
     source: {type: 'device', driverId: 'TrueMetrix'},
     enabled: {mac: true, win: true, linux: true}
   },
+  weitai: {
+    instructions: 'Connect your Equil to your computer',
+    name: 'Equil',
+    image: {
+      src: weitaipump,
+      height: 200,
+      width: 200,
+      alt: 'Weitai link',
+    },
+    key: 'Weitai',
```

Comment:
```suggestion
    key: 'weitai',
```

Reply:
Also, add

```js
Weitai: usbDevice,
```
to `device.DeviceComms` object (line 113 in `lib/core/device.js`)